### ISSUE TITLE
[Accessibility related bug] Wrong accessible name information in main search field because of icon

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2627]
[This is a ticket for the "accessibility backlog"]

<img width="288" alt="Main search form with magnifying glass icon on the right side" src="https://user-images.githubusercontent.com/505181/70324134-f9661300-182e-11ea-9ab5-d541ab9f0a6f.png">

As of now, at least in VoiceOver screen reader the icon inside the main search interferes with its accessible name. Two measures can remedy this:

* Remove `aria-label` from the icon. To achieve consistent screen reader support elements with aria-label must have a role that is not `presentation` (and that is what a `<span>` element has). If said span should serve as a meaningful image it should have a role of `img`. 
* But in this case the magnifying glass icon is purely decorative. Thus my recommendation is to hide it with `aria-hidden="true"`

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
